sdk/metrics/build.gradle.kts

```diff
@@ -21,6 +24,7 @@ dependencies {
     testAnnotationProcessor("com.google.auto.value:auto-value")
 
     testImplementation(project(":sdk:testing"))
+    testImplementation(project(":sdk:metrics-testing"))
```

Comment:
Nit alphabetical (unless renaming package to `metrics-testing` :P)

Reply:
Fixed